recipes/pytest-pyvista/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pytest-pyvista" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytest_pyvista/pytest_pyvista-{{ version }}.tar.gz
+  sha256: 43e6f0a85b6ff23aff51b7f750528de0fdda6da2011a94b636f05d8498fe38a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - flit >=3.2,<4
+  run:
+    - python >=3.8
+    - pytest >=3.5.0
+    - pyvista
+
+test:
+  imports:
+    - pytest_pyvista  # [not win]
+  commands:
+    - pip check
```

Personally, I'm +1 to adding `# [not win]`. `pytest-pyvista` is primarily envisioned for a Linux environment, so I don't think it needs to be perfected for a Windows environment in my mind.
```suggestion
    - pip check  # [not win]
```

---

It is weird that we get this problem presumably by importing pyvista.  Yet pyvista does not skip any checks here on their recipe.

https://github.com/conda-forge/pyvista-feedstock/blob/cd2b71c6d642a3a745fe6eeaa46a9ba52db711f0/recipe/meta.yaml#L30-L36

---

Also `geovista` has no problem about import `pyvista`. I have no idea why it happens 🤷
https://github.com/conda-forge/geovista-feedstock/blob/89bfd5fe9f8807d9a171d1be3c4a53d61928e5b1/recipe/meta.yaml#L40-L47

---

Both have `yum_requirements.txt`?

---

Oh! Good point. Maybe that is the reason.

---

yum requirements should affect only Linux. That is probably not the reason for it. Most of the time, when pip check fails, it is a problem in the package metadata (a conda dependency or this package) and 90% of the time it is harmless to skip.

---

@ocefpaf Thanks for your comment. We will skip it.